README.md

```diff
@@ -1,4 +1,33 @@
+![Version](https://img.shields.io/badge/version-0.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+[![License: MIT](https://img.shields.io/github/license/1-Platform/one-platform)](https://github.com/1-Platform/one-platform/blob/master/LICENSE)
+
 <p align="center"><img src="https://avatars1.githubusercontent.com/u/58499608?s=100&v=4" alt="One Platform Logo"></p>
 
 # One Platform
```

Comment:
Move these lines to the top of the file.

The shields/badges should be below the heading.